django/forms/models.py

```diff
@@ -93,10 +93,17 @@ def model_to_dict(instance, fields=None, exclude=None):
     return data
 
 
+def formfield_apply_limit_choices_to(formfield):
```

Comment:
I avoided this originally due to the "to to". But if it is agreeable, I'll happily go with it.